draft-ietf-tls-tls13.md

```diff
@@ -1817,6 +1817,17 @@ the 0-RTT data without detection. If the client's ephemeral share
 is compromised and client authentication is used, then the attacker
 can impersonate the client on subsequent connections.
 
+It is up to the implementation and application layers above TLS to mitigate
+the impact of these weaknesses.  For example, to support strong client
+authentication and forward secrecy over 0-RTT connections, it is possible to
+issue a new ticket on the first 1-RTT connection, and on every resumed session
+after that.  Each co-located group of servers could ensure that tickets they
+issue are used only once by maintaining a local cache of ticket decryption
+keys.  An accurate clock on the client could ensure freshness of client
```

Comment:
This suggestion has some problems: 

it assumes that there is exactly one connection at any point in time, and not multiple 0-RTT connections established based on a single 1-RTT connection (which is highly likely.

The tickets need to be perfectly atomic and non-portable.

I'd rather that you didn't attempt to give advice like this.  This is very hard to get right.  It's better to say that it is hard and to concentrate on precisely characterizing the problem (as the existing text does).

Note that strong client authentication and forward secrecy are possible on 0-RTT connections, but both require an absolute guarantee of single-use.  If you insist on describing a "solution" to the limitations, you need to be very clear what conditions it to be a genuine solution.  Either way, attempting to describe a mechanism in even as much detail as you do is very risky.